From 6e28f75e7dd4c9136060158f38c6c8d8ec1bca7d Mon Sep 17 00:00:00 2001
From: Samuel Martin <s.martin49@gmail.com>
Date: Mon, 5 May 2014 14:02:18 +0200
Subject: [PATCH 1/3] dynamicEDT3D: install CMake module under
 share/dynamicEDT3D/

dynamicEDT3D was the only subproject installing its CMake module files
under lib/cmake/.
This patch cleans this up, making dynamicEDT3D installation consistent with
the octomap and octovis ones.

Signed-off-by: Samuel Martin <s.martin49@gmail.com>
---
 dynamicEDT3D/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dynamicEDT3D/CMakeLists.txt b/dynamicEDT3D/CMakeLists.txt
index ac24b80..4e7d96a 100644
--- a/dynamicEDT3D/CMakeLists.txt
+++ b/dynamicEDT3D/CMakeLists.txt
@@ -94,7 +94,7 @@ configure_file(dynamicEDT3DConfig-version.cmake.in
 install(FILES
   "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig.cmake"
   "${PROJECT_BINARY_DIR}/InstallFiles/dynamicEDT3DConfig-version.cmake" 
-  DESTINATION lib/cmake/dynamicEDT3D/)
+  DESTINATION share/dynamicEDT3D/)
 
 # Write pkgconfig-file:
 include(InstallPkgConfigFile)
-- 
1.9.2

